docs/platforms/javascript/common/configuration/integrations/onuncaughtexception.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: OnUncaughtException
+description: "Registers handlers to capture global uncaught exceptions. (default)"
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.gcp-functions
+  - javascript.koa
+  - javascript.electron
+---
+
+_Import name: `Sentry.onUncaughtExceptionIntegration`_
+
+This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
+
+The `onUncaughtExceptionIntegration` registers handlers to capture global uncaught exceptions that would cause the process to exit.
+
+This integration _does not_ prevent the process from exiting! If yo want to prevent the process from exiting, you should register your own uncaught exception handler and configure `exitEvenIfOtherHandlersAreRegistered: false` in the integration options.
+
+## Options
+
+### `exitEvenIfOtherHandlersAreRegistered`
+
+_Type: `boolean`_
+
+If set to false, the SDK will _not_ exit if we detect that another uncaught exception handler is registered.
+
+### `ignoreIncomingRequests`
+
+_Type: `(url: string) => boolean`_
+
+Allows to define a method to filter out incoming requests based on the URL. If the method returns `true`, the request will be ignored.
+
+
+<PlatformSection notSupported={["javascript.fastify"]}>
+### `tracing`
+
+_Type: `boolean`_
+
+If set to true, spans will be captured for outgoing HTTP requests.
```

Comment:
Again, just checking if this means by default it's set to false and spans aren't captured for outgoing requests.

Also, just checking if "outgoing HTTP requests" is maybe a copy mistake since this section talks about uncaught exceptions?